sherlock/resources/data.json

```diff
@@ -757,6 +757,26 @@
     "username_claimed": "adam",
     "username_unclaimed": "noonewouldeverusethis"
   },
+  "Instagram": {
+    "errorType": "message",
+    "errorMsg": "href=\"/static/bundles/metro/HttpErrorPage.js/",
+    "url": "https://www.instagram.com/{}",
+    "urlMain": "https://www.instagram.com/",
+    "username_claimed": "blue",
+    "username_unclaimed": "noonewouldeverusethis7",
+    "headers": {
```

Comment:
Are you sure that this additional headers are required?

Reply:
@pkubiak right, I checked and headers are not required